chapters/BP_016_en.md

```diff
@@ -35,4 +35,4 @@ Open source software projects are often forked to improve performance. Performan
 
 | The number of ...                                      | is inferior or equals  |  
 |--------------------------------------------------------|:----------------------:|
-| performance-oriented forks is greater than or equal to |           1            |
+| performance-oriented forks are greater than or equal to |           1            |
```

Comment:
the verb is associated with the number : « the number is greated than are equal to »

Reply:
@rlemaire is right.